bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -75,6 +75,11 @@
     protected final static String BOOKIE_ERROR_THRESHOLD_PER_INTERVAL = "bookieErrorThresholdPerInterval";
     protected final static String BOOKIE_QUARANTINE_TIME_SECONDS = "bookieQuarantineTimeSeconds";
 
+    // Bookie info poll interval
+    protected final static String DISK_WEIGHT_BASED_PLACEMENT_ENABLED = "diskWeightBasedPlacementEnabled";
```

Comment:
What is your use case ? Do you want to retrieve the status from all the bookies just once or on a periodic basis ? I'd suggest when you extend the usage of this feature, make the necessary adjustments. You'll have a clearer idea of what the requirements are.